### PR TITLE
Add linter dependency

### PR DESCRIPTION
**Pull request by smashwilson**
Step 1 of #13. This adds the linter as a `devDependency` of both the frontend and backend packages, but does _not_ reformat anything or enforce style in CI.

If you want to preview what it'll look like, run:

```sh
pushd hcfb-fe
npm ci
npm run format
popd

pushd hcfb-be
npm ci
npm run format
popd

# See how it looks in your editor.

git checkout -- .
```

### CODE REVIEW

File: hcfb-fe/tslint.json [hunk 1]
```diff
@@ -6,31 +6,16 @@
     "arrow-return-shorthand": true,
     "callable-types": true,
     "class-name": true,
-    "comment-format": [
-      true,
-      "check-space"
-    ],
-    "curly": true,
```
Comment (smashwilson) [inline hcfb-fe/tslint.json:13]: These `tslint.json` changes are made to keep tslint around for _style_ concerns, but disable it for _formatting_, based on this article:

https://medium.com/@victormejia/setting-up-prettier-in-an-angular-cli-project-2f50c3b9a537

**Comment by bluek1te** [comment]
Am trying to use this for the angular side, but it's only looking for .js files? Is there a flag I need to pass to make it look at .ts files as well?

**Comment by smashwilson** [comment]
No... no, that was just me being thick. I've fixed the path globs in `package.json` _and_ I've set up prettier in the _correct_ `package.json` for the backend. This should be better now?

**Comment by bluek1te** [comment]
This looks great, I wish I knew about this during my senior design project haha. @BenjaminWalkerBond can you load this on the backend and confirm that you're okay with how it formats (+review the pull request)? I think for directory changes, the commands will be:

pushd hcfb-be/backend-server
npm ci
npm run format
popd

**Review by bluek1te — APPROVED** [review]

**Comment by smashwilson** [comment]
Ah, test failures will depend on #23 being merged and conflicts resolved.

@BenjaminWalkerBond was your 👍 reaction intended to be a PR review? For it to count as an "approval," you'll have to navigate to [the "Files changed" tab](https://github.com/bluek1te/feed-txstate/pull/17/files) and click "Review changes". Disregard if you already know how this works and just haven't had a chance to look it over yet 😇

**Comment by bluek1te** [comment]
@holliew @shollens01 @BenjaminWalkerBond Can one of you guys review this please

**Review by BenjaminWalkerBond — APPROVED** [review]
I tested the code in the backend directory and everything looks good.

**Comment by BenjaminWalkerBond** [comment]
Why does the express npm install and build script keep failing? The same thing happened on my "bens_backend_connection" branch...

**Comment by bluek1te** [comment]
Are you talking about the automated build? If that's what you're talking about it's because I moved everything a directory up in #23

**Pull request merged**